stegobox/codec/LSB_wav.py

```diff
@@ -0,0 +1,49 @@
+import random
+import wave
+from typing import NamedTuple, Tuple
+
+from .base import BaseCodec
+
+
+class LSBwav(BaseCodec):
```

Comment:
建议？

```suggestion
class LSBWav(BaseCodec):
```

这个都行，我个人觉得 `Wav` 看着舒服一点。